lib/screens/transfer/scan/interactor/viewdata/network_data.dart

```diff
@@ -0,0 +1,7 @@
+class NetworkData {
```

Comment:
This class was already defined above - we probably do not want to define it twice...